examples/sealevel-curve-extraction.jl

```diff
@@ -0,0 +1,38 @@
+# sealevel curve extraction
+#module Import_SL
+
+using Interpolations
+using DataFrames
+using CSV
+using HTTP
+#import OpenScienceFramework as OSF
+
+function sealevel(url)
+    #=osf = OSF.Client(; token = alexa)  # put your OSF token here
```

Comment:
This actually would work, provided the following:

1. There is a view-only link for download for the project
2. Files are in a folder within the project

For example, this would download the first file in the first folder:
```
import OpenScienceFramework as OSF

osf = OSF.Client(; token="$TOKEN")  
proj = OSF.project(osf; title="CarboKitten")
dirs = readdir(OSF.Directory, proj)

files = readdir(OSF.File, first(dirs))

url = OSF.url(first(files))
download(url)
```
But it does not recover the file name, admittedly, and doesn't solve the problem with using the token.

Reply:
used datahugger and seems work now. However, one of the trials tonight could not properly extract the file, returned an empty file.